inst/stan/estimate_infections.stan

```diff
@@ -139,6 +139,13 @@ transformed parameters {
     }
   }
 
+  // accumulate reports
+  if (any_accumulate) {
+     profile("accumulate") {
+       reports = accumulate_reports(reports, accumulate);
```

Comment:
What is the interaction with truncation here? Before accumulation happened after truncation and now it happens before. I think it could be either  but perhaps after is slightly more common?